source/frontend/src/features/leases/detail/LeasePages/tenant/AddLeaseTenantContainer.tsx

```diff
@@ -11,24 +11,32 @@ import * as React from 'react';
 import { useContext } from 'react';
 import { useState } from 'react';
 
-import AddLeaseTenantForm from './AddLeaseTenantForm';
-import PrimaryContactWarningModal, {
+import { IAddLeaseTenantFormProps } from './AddLeaseTenantForm';
+import {
   getOrgsWithNoPrimaryContact,
+  IPrimaryContactWarningModalProps,
 } from './PrimaryContactWarningModal';
 import { FormTenant } from './ViewTenantForm';
 
 interface IAddLeaseTenantContainerProps {
   formikRef: React.RefObject<FormikProps<IFormLease>>;
   onEdit?: (isEditing: boolean) => void;
+  View: React.FunctionComponent<
+    React.PropsWithChildren<IAddLeaseTenantFormProps & IPrimaryContactWarningModalProps>
+  >;
 }
 
 export const AddLeaseTenantContainer: React.FunctionComponent<
   React.PropsWithChildren<IAddLeaseTenantContainerProps>
-> = ({ formikRef, onEdit, children }) => {
+> = ({ formikRef, onEdit, children, View }) => {
   const { lease, setLease } = useContext(LeaseStateContext);
-  const [selectedTenants, setSelectedTenants] = useState<FormTenant[]>(
+  const [tenants, setTenants] = useState<FormTenant[]>(
     apiLeaseToFormLease(lease as ILease)?.tenants || [],
   );
+  const [selectedContacts, setSelectedContacts] = useState<IContactSearchResult[]>(
+    apiLeaseToFormLease(lease as ILease)?.tenants.map(t => t.toContactSearchResult()) || [],
```

Comment:
Why is the casting necessary?

Reply:
will check

Reply:
its not, maybe copied this from somewhere? removed.